server/src/optima/dbmodels.py

```diff
@@ -46,7 +46,9 @@ class ProjectDb(db.Model):
                                 uselist=False)
     project_data = db.relationship('ProjectDataDb', backref='projects',
                                 uselist=False)
-    creation_time = db.Column(db.DateTime(timezone=True), server_default=text('now()'))
+    creation_time = db.Column(db.DateTime(timezone=True), default=db.func.now())
```

Comment:
Regarding the difference between default and server_default (which we consistently have in many places already :) )  server_default means you don't provide the value to the column and DB fills the default there; default means you specify which default value to provide in 'INSERT' command. I prefer the first approach because it guarantees that there will always be a value if something is inserted.